### PR TITLE
feat(docs): default font - Montserrat

### PR DESCRIPTION
**Pull request by rachelbt**

### CODE REVIEW

**Comment by yinonov** [comment]
denote https://github.com/Vonage/spezia-webfont-kit

File: apps/docs/assets/styles/iframe.css [hunk 1]
```diff
@@ -1,3 +1,10 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@100;300;400;500;600;700&display=swap');
```
Comment (yinonov) [inline apps/docs/assets/styles/iframe.css:1]: document the need of this

File: apps/docs/assets/styles/iframe.css [hunk 2]
```diff
@@ -1,3 +1,10 @@
...
+
+:root {
+	--vvd-font-family-upright: Montserrat;
```
Comment (yinonov) [inline apps/docs/assets/styles/iframe.css:4]: no need. will be specified in tokens

**Review by yinonov — CHANGES_REQUESTED** [review]

File: apps/docs/assets/styles/iframe.css (hunk 2 above)
Comment (rachelbt) [reply apps/docs/assets/styles/iframe.css:4]: sure. but for preview we still need it until we change the token.
Leaving if till we choose a font for monospace as well...

**Comment by rachelbt** [comment]
@yinonov tried using [fira-code font](https://fonts.google.com/specimen/Fira+Code?query=fira) but it didn't work good with Montserrat:
image

added [Roboto Mono](https://fonts.google.com/specimen/Roboto+Mono?query=roboto).

I'm wondering if its not bad for performance to load 2 fonts.
If we will choose to leave it with 2 fonts - I'll change  the import statement to include them both as Google-Fons provides.

**Comment by yinonov** [comment]
> @yinonov tried using [fira-code font](https://fonts.google.com/specimen/Fira+Code?query=fira) but it didn't work good with Montserrat: image
> 
> added [Roboto Mono](https://fonts.google.com/specimen/Roboto+Mono?query=roboto).
> 
> I'm wondering if its not bad for performance to load 2 fonts. If we will choose to leave it with 2 fonts - I'll change the import statement to include them both as Google-Fons provides.

fonts aren't fetched until actually used. therefore the FOUT. you should include them both in the css call to google fonts. notice in the network pane in dev-tools they won't load unless actually use in page

File: apps/docs/_layouts/base.njk [hunk 3]
```diff
@@ -19,13 +19,14 @@
...
-
-    <link rel="stylesheet" href="{{ '/assets/styles/fonts/spezia.css' | url }}">
+		<!-- Load fonts -->
+		<link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@100;300;400;500;600;700&display=swap" rel="stylesheet">
+
+		<link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@400&display=swap" rel="stylesheet">
```
Comment (yinonov) [inline apps/docs/_layouts/base.njk:29]: why are they split? can be reduced to a 1 call

**Review by yinonov — CHANGES_REQUESTED** [review]

File: apps/docs/assets/styles/iframe.css [hunk 4]
```diff
@@ -1,3 +1,15 @@
+/*font need to be imported or via link in head or in the css
+in the iframe we need this to effect the components preview
+*/
+
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@200;300;400;500;600;700;800;900&display=swap');
```
Comment (yinonov) [inline apps/docs/assets/styles/iframe.css:5]: do we need all these weights?

**Review by yinonov — CHANGES_REQUESTED** [review]

File: apps/docs/assets/styles/main.css [hunk 5]
```diff
@@ -1,4 +1,8 @@
 /* #region reset */
+:root {
+	--vvd-font-family-upright: Montserrat;
+	--vvd-font-family-monospace: Roboto Mono;
```
Comment (yinonov) [inline apps/docs/assets/styles/main.css:4]: that's where we override font families to Spezia

**Review by yinonov — APPROVED** [review]

**Comment by rachelbt** [comment]
tokens for default font.
Basically I removed the `font-stretch` properties.

looks quit the same :)

image


```
--vvd-typography-headline: 500  calc(var(--vvd-size-font-scale-base, 16px) * 4.125)/1.3333333333333333 var(--vvd-font-family-upright, Montserrat);
--vvd-typography-subtitle: 500  calc(var(--vvd-size-font-scale-base, 16px) * 3.25)/1.3076923076923077 var(--vvd-font-family-upright, Montserrat);
--vvd-typography-heading-1: 500  calc(var(--vvd-size-font-scale-base, 16px) * 2.5)/1.3 var(--vvd-font-family-upright, Montserrat);
--vvd-typography-heading-2: 500  calc(var(--vvd-size-font-scale-base, 16px) * 2)/1.375 var(--vvd-font-family-upright, Montserrat);
--vvd-typography-heading-3: 500  calc(var(--vvd-size-font-scale-base, 16px) * 1.625)/1.3846153846153846 var(--vvd-font-family-upright, Montserrat);
--vvd-typography-heading-4: 500  calc(var(--vvd-size-font-scale-base, 16px) * 1.25)/1.4 var(--vvd-font-family-upright, Montserrat);
--vvd-typography-base: 400  calc(var(--vvd-size-font-scale-base, 16px) * 0.875)/1.4285714285714286 var(--vvd-font-family-upright, Montserrat);
--vvd-typography-base-bold: 600  calc(var(--vvd-size-font-scale-base, 16px) * 0.875)/1.4285714285714286 var(--vvd-font-family-upright, Montserrat);
--vvd-typography-base-code: 400  calc(var(--vvd-size-font-scale-base, 16px) * 0.875)/1.4285714285714286 var(--vvd-font-family-monospace, Roboto Mono);
--vvd-typography-base-condensed: 400  calc(var(--vvd-size-font-scale-base, 16px) * 0.75)/1.3333333333333333 var(--vvd-font-family-upright, Montserrat);
--vvd-typography-base-condensed-bold: 600  calc(var(--vvd-size-font-scale-base, 16px) * 0.75)/1.3333333333333333 var(--vvd-font-family-upright, Montserrat);
--vvd-typography-base-condensed-code: 400  calc(var(--vvd-size-font-scale-base, 16px) * 0.75)/1.3333333333333333 var(--vvd-font-family-monospace, Roboto Mono);
--vvd-typography-base-extended: 400  calc(var(--vvd-size-font-scale-base, 16px))/1.5 var(--vvd-font-family-upright, Montserrat);
--vvd-typography-base-extended-bold: 600  calc(var(--vvd-size-font-scale-base, 16px))/1.5 var(--vvd-font-family-upright, Montserrat);
--vvd-typography-base-extended-code: 400  calc(var(--vvd-size-font-scale-base, 16px))/1.5 var(--vvd-font-family-monospace, Roboto Mono);
```

File: apps/docs/_layouts/base.njk [hunk 6]
```diff
@@ -19,13 +19,14 @@
...
 
-    <!-- Load fonts -->
-    <link rel="preload" href="/assets/styles/fonts/SpeziaCompleteVariableItalicWeb.woff2" as="font" type="font/woff2" crossorigin>
-    <link rel="preload" href="/assets/styles/fonts/SpeziaCompleteVariableUprightWeb.woff2" as="font" type="font/woff2" crossorigin>
-    <link rel="preload" href="/assets/styles/fonts/SpeziaMonoCompleteVariableWeb.woff2" as="font" type="font/woff2" crossorigin>
-
-    <link rel="stylesheet" href="{{ '/assets/styles/fonts/spezia.css' | url }}">
+		<link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@400&display=swap" rel="stylesheet">
```
Comment (yinonov) [inline apps/docs/_layouts/base.njk:29]: we don't really need this, rather just document it

File: apps/docs/assets/styles/iframe.css [hunk 7]
```diff
@@ -1,3 +1,15 @@
...
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@400&display=swap');
+
+:root {
+	--vvd-font-family-upright: Montserrat;
+	--vvd-font-family-monospace: Roboto Mono;
+}
```
Comment (yinonov) [inline apps/docs/assets/styles/iframe.css:11]: not actually needed

File: apps/docs/assets/styles/spezia.css [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+	--vvd-typography-base-condensed: 400 ultra-condensed calc(var(--vvd-size-font-scale-base, 16px) * 0.75)/1.3333333333333333 var(--vvd-font-family-upright, SpeziaCompleteVariableUpright);
+	--vvd-typography-base-condensed-bold: 600 ultra-condensed calc(var(--vvd-size-font-scale-base, 16px) * 0.75)/1.3333333333333333 var(--vvd-font-family-upright, SpeziaCompleteVariableUpright);
+	--vvd-typography-base-condensed-code: 400 ultra-condensed calc(var(--vvd-size-font-scale-base, 16px) * 0.75)/1.3333333333333333 var(--vvd-font-family-monospace, SpeziaMonoCompleteVariable);
+	--vvd-typography-base-extended: 400 ultra-condensed calc(var(--vvd-size-font-scale-base, 16px))/1.5 var(--vvd-font-family-upright, SpeziaCompleteVariableUpright);
+	--vvd-typography-base-extended-bold: 600 ultra-condensed calc(var(--vvd-size-font-scale-base, 16px))/1.5 var(--vvd-font-family-upright, SpeziaCompleteVariableUpright);
+	--vvd-typography-base-extended-code: 400 ultra-condensed calc(var(--vvd-size-font-scale-base, 16px))/1.5 var(--vvd-font-family-monospace, SpeziaMonoCompleteVariable);
```
Comment (yinonov) [inline apps/docs/assets/styles/spezia.css:40]: now on 2nd thought, if it's already a requirement to override tokens (due to stretch value), why should the font family be a public variable? it's half of a solution. requires a better API strategy

**Review by yinonov — CHANGES_REQUESTED** [review]
consider some of the comments are just a "note to self"

File: libs/components/README.md [hunk 9]
```diff
@@ -152,8 +151,9 @@ Tools like [UNPKG](https://unpkg.com), [jsDeliver](https://www.jsdelivr.com), [S
 The following snippet fully renders a Vivid button component
 
 ```html
-<!-- import spezia font -->
-<link rel="stylesheet" href="https://unpkg.com/@vonage/vivid@next/styles/fonts/spezia.css">
+<!-- import Montserrat font -->
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;600&display=swap" rel="stylesheet">">
```
Comment (yinonov) [inline libs/components/README.md:156]: what about roboto?

**Review by yinonov — CHANGES_REQUESTED** [review]

File: libs/components/README.md (hunk 9 above)
Comment (yinonov) [reply libs/components/README.md:156]: oh not needed for button?

File: libs/components/README.md [hunk 10]
```diff
@@ -47,7 +47,7 @@ Tokens folder contains the following files:
 
 Only one theme is required to be loaded.
 
-### Fonts (Prerequisite)
+### Fonts (Prerequisite) - Should we leave it for Vonage users?
```
Comment (yinonov) [inline libs/components/README.md:50]: explain how to import the font

**Review by yinonov — CHANGES_REQUESTED** [review]

File: libs/components/README.md (hunk 9 above)
Comment (rachelbt) [reply libs/components/README.md]: it is not needed for the button - but maybe its better we add it in the import statement so it will not be forgotten.

Comment (yinonov) [reply libs/components/README.md]: it can be imported when set in the query parameter together with `Montserrat` right?

Comment (rachelbt) [reply libs/components/README.md]: Yes - was working on it - didn't merged yet

**Review by yinonov — APPROVED** [review]

**Pull request merged**